manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```
@@ -17,6 +17,8 @@ data:
   mlmdDb: metadb
   cacheDb: cachedb
   pipelineDb: mlpipeline
+  objectStoreHost: minio-service
+  objectStorePort: 9000
```

```suggestion
  objectStorePort: "9000"
```
env variable has to be string, not number.